docs/examples/setting-the-language.md

```diff
@@ -0,0 +1,52 @@
+---
+description: |
+  Set the lang attribute in the <html> tag.
+---
+
+When you initialize a project with `deno run -A -r https://fresh.deno.dev`,
```

Comment:
Nit: The directory argument is missing. Running this command as is will lead to an error. I'm hoping to make it optional with #1263 , but that's not merged yet.

Reply:
Yes, I suppose I should have mentioned that when I opened this. I left it off, since I saw your 1263 and I figured the shortened version was clear enough. Perhaps this can be merged once the other one is merged.

Reply:
Discard my comment. #1263 is merged 🎉